RELEASENOTES.md

```diff
@@ -20,14 +20,18 @@
 -->
 # Release Notes
 
+
+### 4.0.4 (Oct 02, 2017)
```

Comment:
Please do not edit RELEASENOTES.md, plugin.xml or package.json in the PR.  Only the PMC/Committers determine when this is release and what the version number is, and even then it has to be voted on by the Committers/PMC.

Reply:
Oh okay sorry for the edit, i'll remember for the next time.